resip/dum/SubscriptionCreator.cxx

```diff
@@ -15,6 +15,7 @@ SubscriptionCreator::SubscriptionCreator(DialogUsageManager& dum,
 
    mLastRequest->header(h_Event).value() = event; 
    mLastRequest->header(h_Expires).value() = subscriptionTime;
+   mLastRequest->header(h_Accepts).push_back( Mime( "application","pidf+xml") );
```

Comment:
@MateusBellomo remove the change from ```SubscriptionCreator``` and try using ```addSupportedMimeType(NOTIFY, Pidf::getStaticType())``` on the ```MasterProfile``` in your ```apps/telepathy``` code in pull request #53.  Does that sound correct @sgodin ?